src/styles/index.scss

```diff
@@ -0,0 +1,40 @@
+@import 'variables';
+@import 'card';
+@import 'stars';
+@import 'header';
+
+* {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.main {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 48px 46px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .main {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    --column-count: 4;
+  }
```

Comment:
use scss nesting

Reply:
Agreed with prev comment. You can find an example in comment in stars.scss file.